x-pack/packages/kbn-elastic-assistant/impl/assistant/index.tsx

```diff
@@ -352,7 +205,12 @@ const AssistantComponent: React.FC<Props> = ({
   // Show missing connector callout if no connectors are configured
 
   const showMissingConnectorCallout = useMemo(() => {
-    if (!isLoading && areConnectorsFetched && currentConversation?.id !== '') {
+    if (
+      !isLoadingCurrentUserConversations &&
+      isFetchedConnectors &&
+      currentConversation &&
```

Comment:
Adding this condition skips a flash of the missing connector callout when the current conversation is still loading.